sphinx_gherkindoc/writer.py

```diff
@@ -193,7 +193,9 @@ def steps(steps: List[behave.model.Step]) -> None:
                 step.line,
             )
             bold_step = re.sub(r"(\\\<.*?\>)", r"**\1**", rst_escape(step.name))
-            step_text = u"- {} {}".format(_format_keyword(step.keyword), bold_step)
+            step_text = u"| - {} {}".format(_format_keyword(step.keyword), bold_step)
+            if step.keyword in ADD_ON_STEP_KEYWORDS:
+                step_text = step_text.replace("| -", "| ")
```

Comment:
If we decide to go this way, you could just make this:
```
prefix = "|  " if step.keyword in ADD... else "| -"
```
and then just `.format` it in place.
Very minor thang, just popped out at me. :-)

Reply:
Yeah definitely I'm sure there's a nicer way to write this. Once we get a visual we like, I'll clean up the code.